File: Pagina_web/Entity/fotoInmueble.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Entity
{
    public class fotoInmueble
    {
        [Key]
        [Column(TypeName = "nvarchar(11)")]
        public String Codigo { get; set; }
        [Column(TypeName = "nvarchar(11)")]
        public String CodInmueble { get; set; }
        public byte[] Imagen { get; set; }
    }
}

File: Aplicativo_Movil/app/src/main/java/com/naacdeveloper/leco/FotoInmueble.kt
package com.naacdeveloper.leco

class FotoInmueble (codigo: String, codInmueble: String, imagen: String ){
    var codigo: String = "";
    var codInmueble: String = "";
    var imagen: String = "";

    init {
        this.codInmueble = codInmueble;
        this.codigo = codigo;
        this.imagen = imagen;
    }
}